src/Sarif.Driver/Sdk/ExitReason.cs

```diff
@@ -11,6 +11,7 @@ public enum ExitReason
         UnhandledExceptionInstantiatingSkimmers,
         UnhandledExceptionInEngine,
         NoRulesLoaded,
+        IncompatibleRulesDetected,
```

Comment:
Good catch! We need this value to be appended to the end of the enum.

Alternately, if you wanted to preserve some specific order, you could explicitly define the value of each enum. 

If you simply add the value to the end, please add this comment at the top of the class.

// Be sure to append new values for this enum to avoid changing existing member values.

Reply:
move the new enum to the end. comment added.